### PR TITLE
Link to subscriptions migration steps

### LINKED ISSUE #6323: the migration guide to graphql-ws doesn't provide guidance or useful info
subscriptions-transport-ws is deprecated the npm page links to an apollographql "migration guide" that doesn't contain any useful information,

https://www.npmjs.com/package/subscriptions-transport-ws

https://www.apollographql.com/docs/apollo-server/data/subscriptions/#switching-from-subscriptions-transport-ws

please provide an upgrade path for users of subscriptions-transport-ws

### PR DESCRIPTION
**Pull request by trevor-scheer**
Fixes #6323

### CODE REVIEW

File: docs/source/data/subscriptions.mdx [hunk 1]
```diff
@@ -572,7 +572,7 @@ If none of these libraries fits your use case, you can also create your own `Pub
 
 >If you use subscriptions with Apollo Client you must ensure **[both your client and server subscription protocols are consistent](/react/data/subscriptions/#choosing-a-subscription-library)** for the subscription library you're using.
 
-This article previously demonstrated using the `subscriptions-transport-ws` library to set up subscriptions. However, this library is no longer actively maintained. You can still use it with Apollo Server, but we strongly recommend using [`graphql-ws`](https://www.npmjs.com/package/graphql-ws) instead.
+This article previously demonstrated using the `subscriptions-transport-ws` library to set up subscriptions. However, this library is no longer actively maintained. You can still use it with Apollo Server, but we strongly recommend using [`graphql-ws`](https://www.npmjs.com/package/graphql-ws) instead. A migration path from `subscriptions-transport-ws` to `graphql-ws` is outlined in the [Apollo Server v3 migration guide](/apollo-server/migration/#subscriptions)
```
Comment (trevor-scheer) [inline docs/source/data/subscriptions.mdx:575]:
```suggestion
This article previously demonstrated using the `subscriptions-transport-ws` library to set up subscriptions. However, this library is no longer actively maintained. You can still use it with Apollo Server, but we strongly recommend using [`graphql-ws`](https://www.npmjs.com/package/graphql-ws) instead. A migration path from `subscriptions-transport-ws` to `graphql-ws` is outlined in the [Apollo Server v3 migration guide](/apollo-server/migration/#subscriptions).
```

**Comment by trevor-scheer** [comment]
I think I pushed a broken link but netlify isn't telling me anything is wrong. @trevorblades could you take a look when you get a chance please?

File: docs/source/data/subscriptions.mdx [hunk 2]
```diff
@@ -572,7 +572,7 @@ If none of these libraries fits your use case, you can also create your own `Pub
 
 >If you use subscriptions with Apollo Client you must ensure **[both your client and server subscription protocols are consistent](/react/data/subscriptions/#choosing-a-subscription-library)** for the subscription library you're using.
 
-This article previously demonstrated using the `subscriptions-transport-ws` library to set up subscriptions. However, this library is no longer actively maintained. You can still use it with Apollo Server, but we strongly recommend using [`graphql-ws`](https://www.npmjs.com/package/graphql-ws) instead.
+This article previously demonstrated using the `subscriptions-transport-ws` library to set up subscriptions. However, this library is no longer actively maintained. You can still use it with Apollo Server, but we strongly recommend using [`graphql-ws`](https://www.npmjs.com/package/graphql-ws) instead. A migration path from `subscriptions-transport-ws` to `graphql-ws` is outlined in the [Apollo Server v3 migration guide](/apollo-server/migration/#subscriptions).
```
Comment (trevor-scheer) [inline docs/source/data/subscriptions.mdx:575]:
```suggestion
This article previously demonstrated using the `subscriptions-transport-ws` library to set up subscriptions. However, this library is no longer actively maintained. You can still use it with Apollo Server, but we strongly recommend using [`graphql-ws`](https://www.npmjs.com/package/graphql-ws) instead. A migration path from `subscriptions-transport-ws` to `graphql-ws` is outlined in the [Apollo Server v3 migration guide](/migration/#subscriptions).
```

**Comment by trevor-scheer** [comment]
@rkoron007 has brought it to my attention that this is in fact _not_ a migration guide that I attempted to link to. She's graciously agreed to put an actual one together next week! cc @iambumblehead

**Comment by iambumblehead** [comment]
@trevor-scheer thank you :)

**Pull request merged**